CHANGELOG.rst

```diff
@@ -6,6 +6,97 @@ Changelog
 
     The Dutch version of this changelog can be found :ref:`here <changelog-nl>`.
 
+3.4.0-alpha.2 (2025-12-04)
+==========================
+
+This is an alpha release, meaning it is not finished yet or suitable for production use.
+
+Upgrade procedure
+-----------------
+
+To upgrade to 3.4, please:
+
+* ⚠️ Ensure you are currently on Open Forms 3.2.x or newer.
+* if you've built custom NL DS themes, please review the deprecations in the SDK
+  changelog.
+
+.. warning:: If you've built dashboards using the metrics telemetry, you'll need to
+   update them with the updated metric names. The new names are in the documentation.
+
+New features
+------------
+
+* [:backend:`5704`] You can now filter forms in the admin based on the payment backend
+  used.
+* [:backend:`5643`] Added ``heeftAlsAanspreekpunt`` to the StUF-ZDS registration plugin.
+* [:backend:`5707`] Implemented prefill plugin to retrieve the preferences for the
+  customer profile from customer interaction APIs like Open Klant.
+* Included the latest SDK with the latest (beta version of the) renderer. We'd like
+  everyone to actively test this now. Some known limitations at this point:
+
+  * the map component integration is not available yet.
+  * the customer profile component integration is not available yet.
+  * the datepicker (``date`` and ``datetime`` components) has known (UX) issues.
+  * themes may be broken due to missing design token values in custom themes.
```

Comment:
This is a problem on their end right?

Reply:
yes and no - we need to add the backwards compatibility layers and document the necessary changes/deprecations :) will be done for the full release